src/elli/importer/accurion.py

```diff
@@ -0,0 +1,99 @@
+"""A helper class to load data from Accurion EP4 DAT files.
+Typical files look like: Si3N4_on_4inBF33_W03_20240905-105631.ds.dat
+"""
+
+import pandas as pd
+import chardet
```

Comment:
This needs to be specified in our `pyproject.toml` dependencies if we explicitly use it here. It could be a good addition also to the other ASCII based readers we have. I didn't know this library before. Thanks for showing it to me :)